### PR TITLE
Release 3.0.3

### PR DESCRIPTION
**Pull request by miaulalala**

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -32,7 +36,8 @@
 	<screenshot>https://raw.githubusercontent.com/nextcloud/calendar/master/screenshots/week_room_suggestion.png</screenshot>
 	<screenshot>https://raw.githubusercontent.com/nextcloud/calendar/master/screenshots/week_sidebar.png</screenshot>
 	<dependencies>
-		<nextcloud min-version="20" max-version="24" />
+		<php min-version="7.3" max-version="8.0" />
+		<nextcloud min-version="21" max-version="24" />
```
Comment (ChristophWurst) [inline appinfo/info.xml:40]: :-1: revert, we don't drop Nextcloud 20 in a patch release

File: appinfo/info.xml [hunk 2]
```diff
@@ -32,7 +36,8 @@
 	<screenshot>https://raw.githubusercontent.com/nextcloud/calendar/master/screenshots/week_room_suggestion.png</screenshot>
 	<screenshot>https://raw.githubusercontent.com/nextcloud/calendar/master/screenshots/week_sidebar.png</screenshot>
 	<dependencies>
-		<nextcloud min-version="20" max-version="24" />
+		<php min-version="7.3" max-version="8.0" />
```
Comment (ChristophWurst) [inline appinfo/info.xml:39]: same here, 7.2 is still supported for 3.0.x

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Wait – the package* changes are suspicious

**Comment by ChristophWurst** [comment]
edit only the version number in package.json and package-lock.json :)

**Comment by miaulalala** [comment]
yeah i need to start over, i made a mess

**Pull request closed without merging**